assets/queries/terraform/azure/email_alerts_are_disabled/query.rego

```diff
@@ -0,0 +1,30 @@
+package Cx
+
+CxPolicy[result] {
+	resource := input.document[i].resource.azurerm_security_center_contact[name]
+
+    object.get(resource,"alert_notifications","undefined") != "undefined"
+	not resource.alert_notifications
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("azurerm_security_center_contact[%s].alert_notifications", [name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": sprintf("'azurerm_security_center_contact.%s.alert_notifications' is true", [name]),
+		"keyActualValue": sprintf("'azurerm_security_center_contact.%s.alert_notifications' is false", [name]),
+	}
+}
+
+CxPolicy[result] {
+	resource := input.document[i].resource.azurerm_security_center_contact[name]
+
+    object.get(resource,"alert_notifications","undefined") == "undefined"
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("azurerm_security_center_contact[%s]", [name]),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": sprintf("'azurerm_security_center_contact.%s.alert_notifications' is set", [name]),
+		"keyActualValue": sprintf("'azurerm_security_center_contact.%s.alert_notifications' is undefined", [name]),
+	}
+}
```

Comment:
According to [Terraform documentation](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/security_center_contact), the `alert_notifications` field is required, so you don't need to check if it's undefined

Reply:
Addressed requests